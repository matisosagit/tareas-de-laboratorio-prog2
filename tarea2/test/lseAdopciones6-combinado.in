crearLSEAdopcionesVacia

crearFecha 18/08/1930
crearPerro 1 Pluto 6 8 Perrito simpaticon
crearPersona 12345678 Mickey Mouse 18 11 1928
agregarPerroPersona
crearFecha 2/1/2025
insertarLSEAdopciones 

crearFecha 13/09/1969
crearPerro 2 Scooby 3 9 Perrito detective
crearPersona 23456789 Shaggy Rogers 25 02 1998
agregarPerroPersona
crearFecha 1/1/2025
insertarLSEAdopciones

crearFecha 04/10/1950
crearPerro 3 Snoopy 5 7 Perrito campechano
crearPersona 34567890 Charlie Brown 30 10 1950
agregarPerroPersona
crearFecha 4/1/2025
insertarLSEAdopciones

crearFecha 17/04/2011
crearPerro 4 Fantasma 2 3 Perrito blanco
crearPersona 45678901 Jon Snow 26 12 1986
agregarPerroPersona
crearFecha 3/1/2025
insertarLSEAdopciones

crearFecha 22/06/1955
crearPerro 5 Reina 7 8 Perrita dama
crearPersona 56789012 Jim Dear 10 03 1927
agregarPerroPersona
liberarPerro
crearFecha 22/06/1955
crearPerro 6 Golfo 8 4 Perrito vagabundo
agregarPerroPersona
crearFecha 3/1/2025
insertarLSEAdopciones

crearFecha 22/06/1955
crearPerro 6 Golfo 8 4 Perrito vagabundo
crearPersona 56789012 Jim Dear 10 03 1927
agregarPerroPersona
liberarPerro
crearFecha 22/06/1955
crearPerro 5 Reina 7 8 Perrita dama
agregarPerroPersona
crearFecha 3/1/2025
insertarLSEAdopciones   
imprimirLSEAdopciones

removerAdopcionLSEAdopciones 12345678 1
removerAdopcionLSEAdopciones 56789012 6
esVaciaLSEAdopciones

crearFecha 16/11/2001
crearPerro 7 Fang 2 3 Perrito guardian
crearPersona 67890123 Rubeus Hagrid 06 12 1928
agregarPerroPersona
crearFecha 1/1/2025
insertarLSEAdopciones

crearFecha 21/11/2008
crearPerro 8 Bolt 4 8 Perrito rapido
crearPersona 78901234 Penny Forrester 20 01 2001
agregarPerroPersona
crearFecha 4/1/2025
insertarLSEAdopciones

existeAdopcionLSEAdopciones 56789012 6
existeAdopcionLSEAdopciones 56789012 5
existeAdopcionLSEAdopciones 67890123 7

esVaciaLSEAdopciones
imprimirLSEAdopciones
liberarLSEAdopciones
Fin
