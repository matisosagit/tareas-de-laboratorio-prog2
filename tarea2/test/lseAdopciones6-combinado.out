1>2>3>4>5>6>7>Adopcion de persona de CI 12345678 y perro de id 1 agregada de forma exitosa.
8>9>10>11>12>13>Adopcion de persona de CI 23456789 y perro de id 2 agregada de forma exitosa.
14>15>16>17>18>19>Adopcion de persona de CI 34567890 y perro de id 3 agregada de forma exitosa.
20>21>22>23>24>25>Adopcion de persona de CI 45678901 y perro de id 4 agregada de forma exitosa.
26>27>28>29>30>31>32>33>34>35>Adopcion de persona de CI 56789012 y perro de id 6 agregada de forma exitosa.
36>37>38>39>40>41>42>43>44>45>Adopcion de persona de CI 56789012 y perro de id 5 agregada de forma exitosa.
46>Lista de adopciones:
---------------------------
Adopcion en fecha 1/1/2025
Adoptante:
Persona Shaggy Rogers
CI: 23456789
Adoptado:
Perro 2
Nombre: Scooby
Fecha de ingreso: 13/9/1969
---------------------------
---------------------------
Adopcion en fecha 2/1/2025
Adoptante:
Persona Mickey Mouse
CI: 12345678
Adoptado:
Perro 1
Nombre: Pluto
Fecha de ingreso: 18/8/1930
---------------------------
---------------------------
Adopcion en fecha 3/1/2025
Adoptante:
Persona Jon Snow
CI: 45678901
Adoptado:
Perro 4
Nombre: Fantasma
Fecha de ingreso: 17/4/2011
---------------------------
---------------------------
Adopcion en fecha 3/1/2025
Adoptante:
Persona Jim Dear
CI: 56789012
Adoptado:
Perro 6
Nombre: Golfo
Fecha de ingreso: 22/6/1955
---------------------------
---------------------------
Adopcion en fecha 3/1/2025
Adoptante:
Persona Jim Dear
CI: 56789012
Adoptado:
Perro 5
Nombre: Reina
Fecha de ingreso: 22/6/1955
---------------------------
---------------------------
Adopcion en fecha 4/1/2025
Adoptante:
Persona Charlie Brown
CI: 34567890
Adoptado:
Perro 3
Nombre: Snoopy
Fecha de ingreso: 4/10/1950
---------------------------
Fin lista de adopciones.
47>Adopcion de la persona de CI 12345678 al perro 1 removida de la lista.
48>Adopcion de la persona de CI 56789012 al perro 6 removida de la lista.
49>La lista de adopciones NO es vacia.
50>51>52>53>54>55>Adopcion de persona de CI 67890123 y perro de id 7 agregada de forma exitosa.
56>57>58>59>60>61>Adopcion de persona de CI 78901234 y perro de id 8 agregada de forma exitosa.
62>NO Existe una adopcion de la persona de CI 56789012 al perro 6 en la lista.
63>Existe una adopcion de la persona de CI 56789012 al perro 5 en la lista.
64>Existe una adopcion de la persona de CI 67890123 al perro 7 en la lista.
65>La lista de adopciones NO es vacia.
66>Lista de adopciones:
---------------------------
Adopcion en fecha 1/1/2025
Adoptante:
Persona Shaggy Rogers
CI: 23456789
Adoptado:
Perro 2
Nombre: Scooby
Fecha de ingreso: 13/9/1969
---------------------------
---------------------------
Adopcion en fecha 1/1/2025
Adoptante:
Persona Rubeus Hagrid
CI: 67890123
Adoptado:
Perro 7
Nombre: Fang
Fecha de ingreso: 16/11/2001
---------------------------
---------------------------
Adopcion en fecha 3/1/2025
Adoptante:
Persona Jon Snow
CI: 45678901
Adoptado:
Perro 4
Nombre: Fantasma
Fecha de ingreso: 17/4/2011
---------------------------
---------------------------
Adopcion en fecha 3/1/2025
Adoptante:
Persona Jim Dear
CI: 56789012
Adoptado:
Perro 5
Nombre: Reina
Fecha de ingreso: 22/6/1955
---------------------------
---------------------------
Adopcion en fecha 4/1/2025
Adoptante:
Persona Charlie Brown
CI: 34567890
Adoptado:
Perro 3
Nombre: Snoopy
Fecha de ingreso: 4/10/1950
---------------------------
---------------------------
Adopcion en fecha 4/1/2025
Adoptante:
Persona Penny Forrester
CI: 78901234
Adoptado:
Perro 8
Nombre: Bolt
Fecha de ingreso: 21/11/2008
---------------------------
Fin lista de adopciones.
67>68>Fin.
